theme/src/components/header.js

```diff
@@ -138,7 +140,11 @@ Header.defaultProps = {
   isSearchEnabled: true
 }
 
-function PrimerNavItems({siteMetadata, items, path}) {
+function PrimerNavItems({siteMetadata, items, path, pathPrefix}) {
+  console.log('header url', siteMetadata.header.url)
+  console.log('pathPrefix', pathPrefix)
+  console.log('path', path)
```

Comment:
Thinking of adding these logs to check values in prod. Not sure if there's a better way to do it 😢